media-sound/teamspeak-server-bin/Manifest

```diff
@@ -1,4 +1,4 @@
-DIST teamspeak3-server_linux_amd64-3.0.13.6.tar.bz2 6492488 BLAKE2B 4cad8545968423c2e65853d73f99a46cbccce628982c0a4418eaf34802d6a171f7e89a4ffc2fbd9fd82afdd9ffcd58d214c894ad88862c2927f1aec72f2d3087 SHA512 95271e5a44c7a14b8fbe134b855973c1af53ac76e6ba01089116b1f63c03bae25f5abfd176f9e26207aa4da8501d646298128078ab5b475458a7ccaba2a48e9c
```

Comment:
You seem to have gotten this Manifest update mixed.

Reply:
Thanks, will be fixed by next commit.